on-prem-docs/content/docs/getting-started.md

```diff
@@ -0,0 +1,541 @@
++++
+title = "Chef Habitat Builder on-prem"
+description = "Chef Habitat Builder overview and pre-requisites"
+draft = false
+bref = ""
+toc = true
++++
+
+**Umbrella Project**: [Chef Habitat](https://github.com/habitat-sh/habitat)
```

Comment:
We can probably remove the meta info - ie, the Umbrella Project, Project State, Issue Response SLA, and Pull Request Response SLA parts

Reply:
Great! I'll leave it on the main readme.